pkgs/tools/networking/openconnect/default.nix

```diff
@@ -1,4 +1,19 @@
-{ lib, stdenv, fetchurl, pkg-config, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib, fetchgit, darwin } :
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, openssl ? null
+, gnutls ? null
+, gmp
+, libxml2
+, stoken
+, zlib
+, fetchgit
+, darwin
+, head ? false
+  , fetchFromGitLab
+  , autoreconfHook
```

Comment:
```suggestion
, head ? false
, fetchFromGitLab
, autoreconfHook
```

Reply:
Again, I use an indent as a cue that these dependencies as conditioned on `head == true`.

Reply:
That will be removed when formatting with either nixpkgs-fmt or nixfmt.